corehq/apps/case_search/management/commands/compare_case_properties.py

```diff
@@ -0,0 +1,85 @@
+from django.core.management.base import BaseCommand
+from corehq.apps.es.case_search import CaseSearchES
+import timeit
+
+
+def define_query(domain, props, operator, case_type=None):
+    script = {
+        "bool": {
+            "must": [
+                {
+                    "script": {
+                        "script": "def val = _source['case_properties'].find{ it -> it.key == '%s' }?.value;"
+                        "_source['case_properties'].find{ it -> it.key == '%s' }?.value %s val;"
+                        % (props[0], props[1], operator),
+                        "lang": "groovy"
+                    }
+                }
+            ]
+        }
+    }
+    if case_type:
```

Comment:
Yes thats exactly what this research is for. Woody mentioned that OBH is interested in incorporating this type of comparison search functionality. The purpose of the script is to get some baseline performance benchmarks because these types of queries require groovy scripting and general feeling is that it is not quite performative.